src/plugin.js

```diff
@@ -72,19 +81,31 @@ function onVjsContextMenu(e) {
     menu.dispose();
   };
 
+  // This is to handle a bug where firefox triggers both 'contextmenu' and 'click'
+  // events on rightclick, causing menu to open and then immediately close.
+  const clickHandler = (_e) => {
```

Comment:
Very minor, but I wonder if this could be renamed to `e`. That's more conventional across our various projects.

Reply:
@misteroneill I would, but `e` is already used in the outer scope (I also get a lint warning about it), Would you prefer a different name instead (perhaps `event`) ?

Reply:
Ah, good point. In that case, I'm fine with `_e`. 👍